test/surface/components/dynamic_component_test.exs

```diff
@@ -0,0 +1,298 @@
+defmodule Surface.Components.ComponentComponentTest do
```

Comment:
```suggestion
defmodule Surface.Components.DynamicComponentTest do
```

Reply:
@Malian good catch! I was also thinking about changing the namespace from `Surface.Components.Component` to `Surface.Components.Dynamic.Component` as we still gonna have to implement a `Surface.Components.Dynamic.LiveComponent` in the near future. Since those built-in components are automatically aliased, I don't mind the larger name. WDYT?